snuba/web/views.py

```diff
@@ -133,12 +133,7 @@ def filter_checked_storages(ignore_experimental: bool) -> List[Storage]:
     for entity in entities:
         entity_storages = entity.get_all_storages()
         for storage in entity_storages:
```

Comment:
nit: Let's refactor this [inline variable](https://refactoring.com/catalog/inlineVariable.html) but otherwise looks good.
```suggestion
        for storage in entity.get_all_storages():
```